src/completion.js

```diff
@@ -144,7 +144,12 @@ module.exports = class KiteCompletionProvider {
     const begin = document.offsetAt(selection.start);
     const end = document.offsetAt(selection.end);
     const enableSnippets = workspace.getConfiguration("kite").enableSnippets;
-    const isSpace = context.triggerCharacter === ' ';
+
+    const isOptionalTrigger = context.triggerCharacter === ' '
```

Comment:
I think this is fine for now, but ideally, we would ensure these triggers are in sync with the triggers in `kite.js`; i.e. enumerating an optional character list, then `completionTriggers` in `kite.js` would concatenate the default and optional triggers.